infrastructure/variables.tf

```diff
@@ -137,6 +137,24 @@ variable "elastic_search_case_index_type" {
   default = "_doc"
 }
 
+variable "elastic_search_nodes_discovery_enabled" {
+  description = "Enable Elasticsearch node discovery by Jest client"
+  type = "string"
+  default = "true"
+}
+
+variable "elastic_search_nodes_discovery_frequency_millis" {
+  description = "Elasticsearch node discovery frequency in milliseconds"
+  type = "string"
+  default = "1000"
```

Comment:
Defaulting discovery check to 1 second. Let me know if this is low or high

Reply:
seems ok to me

Reply:
actually, perhaps a bit too aggressive? how about 30 secs?

Reply:
set to 5 secs